cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/namespace/AbstractStorageProviderNamespaceAdmin.java

```diff
@@ -167,7 +167,14 @@ private void createLocation(NamespaceMeta namespaceMeta) throws IOException {
         }
         // if this is still null at this point, then the directory will have whatever HDFS assigned at creation
         if (groupToSet != null) {
-          namespaceHome.setGroup(groupToSet);
+          try {
```

Comment:
this is an unrelated change - I needed this to fix a unit test failure and I think it is a useful improvement